Lesson22/todo-app/Gokhan/script.js

```diff
@@ -0,0 +1,67 @@
+// const sayHelloBtn = document.getElementById('btnSayHello');
+
+// sayHelloBtn.addEventListener('cli('ck', function (e) {
+//   console.log('Hello', e);
+// });
+
+// const sayByeBtn = document.getElementById('btnSayBye');
+
+// sayByeBtn.addEventListener('click', function () {
+//   console.log('Bye bye');
+// });
+
+const form = document.getElementById('toDoForm');
+
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+
+	// Usually you always need it for form submit events!
+
+	event.preventDefault();
+
+	const input = document.getElementById('toDoInput');
+
+	const listItem = document.createElement('li');
+	listItem.innerText = input.value;
+	listItem.addEventListener('click', function () {
+		listItem.classList.toggle('completed');
```

Comment:
// Here, only toggle the class 'completed' for the span element, not for entire li element
```suggestion
		spanElement.classList.toggle('completed');
```